mobile-app/app/screens/AppNavigator/screens/Dex/components/FavoriteButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { useFavouritePoolpairs } from '@screens/AppNavigator/screens/Dex/hook/FavouritePoolpairs'
+import { ThemedIcon, ThemedTouchableOpacityV2 } from '@components/themed'
+import { tailwind } from '@tailwind'
+
+export function FavoriteButton ({ pairId }: {pairId: string}): JSX.Element {
+  const { isFavouritePoolpair, setFavouritePoolpair } = useFavouritePoolpairs()
+  const isFavouritePair = isFavouritePoolpair(pairId)
+
+  return (
+    <ThemedTouchableOpacityV2
+      onPress={() => setFavouritePoolpair(pairId)}
+      style={tailwind('rounded-full  flex items-center justify-center w-5 h-5')}
+      dark={tailwind({
+        'bg-mono-dark-v2-100': !isFavouritePair,
+        'bg-brand-v2-500': isFavouritePair
+      })}
+      light={tailwind({
+        'bg-mono-light-v2-100': !isFavouritePair,
+        'bg-brand-v2-500': isFavouritePair
+      })}
+    >
+      <ThemedIcon
+        iconType='MaterialIcons'
```

Comment:
i think jonathan just updated the figma to use feather icon

Reply:
it's still a vector . Nothing has changed in Figma

Reply:
check again

Reply:
let's skip this for now, but we can keep this favourite button